Compiler/SimCode/SimCodeMain.mo

```diff
@@ -653,11 +671,19 @@ algorithm
         // Test the parallel code generator in the test suite. Should give decent results given that the task is disk-intensive.
         numThreads := max(1, if Config.getRunningTestsuite() then min(2, System.numProcessors()) else Config.noProc());
         if (not Flags.isSet(Flags.PARALLEL_CODEGEN)) or numThreads==1 then
-          true := max(func() for func in codegenFuncs);
+          res := list(func() for func in codegenFuncs);
         else
-          true := max(l for l in System.launchParallelTasks(numThreads, codegenFuncs, runCodegenFunc));
+          res := System.launchParallelTasks(numThreads, codegenFuncs, runCodegenFunc);
         end if;
-
+        strs := {};
+        for tpl in res loop
+          (true,tmp) := tpl;
+          strs := List.append_reverse(tmp, strs);
+        end for;
+        strs := listReverse(strs);
+        print(stringDelimitList(strs, ", ") + "\n");
```

Comment:
Looks like this line prints the filenames of the partial files, which is not expected output.
@sjoelund Try to comment out!